sdk/keychain/keychain.go

```diff
@@ -126,13 +139,65 @@ func (kc *Keychain) AddLedgerAddresses(addresses []string) error {
 	return fmt.Errorf("keychain is not ledger enabled")
 }
 
-func (kc *Keychain) AddLedgerFunds(amount uint64) error {
+func (kc *Keychain) AddLedgerFunds(
+	network network.Network,
+	amount uint64,
+) error {
 	if kc.LedgerEnabled() {
-		indices, err := kc.Ledger.LedgerDevice.FindFunds(kc.network, amount, 0)
+		indices, err := kc.Ledger.LedgerDevice.FindFunds(network, amount, 0)
 		if err != nil {
 			return err
 		}
 		return kc.AddLedgerIndices(indices)
 	}
 	return fmt.Errorf("keychain is not ledger enabled")
 }
+
+func (kc *Keychain) P(
```

Comment:
can we get address directly from avalanche go Wallet.P() object instead?

Reply:
Seems that it will be something like
```
key := wallet.Keychain.Keychain // This gives you access to the underlying key
pAddress, err := key.P(network.HRP) // network.HRP would be "fuji" for testnet, "avax" for mainnet
```

Reply:
Point of consideration: Should this be in wallet object instead?

Reply:
I believe the account can have it. Not opposed to also have it on wallet.